Source/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -126,6 +126,11 @@ public IFakeOptionsBuilder<T> Implements(Type interfaceType)
                 return this;
             }
 
+            public IFakeOptionsBuilder<T> Implements<TInterface>()
+            {
+                return Implements(typeof (TInterface));

```

Comment:
Hmm. It lives in the `Source\packages`. I thought the warnings would show up anyhow. Huh.
Honestly, I don't really understand the thing. It was in the FakeItEasy when I showed up, and I've never had any experience with it elsewhere.